arch/PKGBUILD

```diff
@@ -0,0 +1,70 @@
+# adapted from https://github.com/archlinux/svntogit-packages/blob/packages/varnish/trunk/PKGBUILD
+
+# before building, be sure to:
+# - replace @VERSION@ with the proper Varnish version to use
+# - run `makepkg -fg >> PKGBUILD` to generate the checksum entries
+
+pkgname=varnish
+pkgver=@VERSION@
+pkgrel=1
+pkgdesc="High-performance HTTP accelerator"
+arch=('x86_64')
+url="https://www.varnish-cache.org/"
+license=('BSD')
+depends=('gcc' 'libnsl' 'pcre2')
+makedepends=('python-docutils' 'python-sphinx' 'git')
+optdepends=('python: needed for vmod development')
+backup=('etc/varnish/default.vcl')
+install=$pkgname.install
+source=("https://varnish-cache.org/_downloads/varnish-$pkgver.tgz"
+	varnishreload
+	varnish.install
+	varnish.logrotate
+	varnish.service
+	varnishncsa.service
+        varnish.sysusers)
+
+prepare() {
+  cd "varnish-$pkgver"
+
+  ./autogen.sh
+}
+
+build() {
+  cd "varnish-$pkgver"
+
+  ./configure \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --localstatedir=/var/lib \
+    --sbindir=/usr/bin
+
+  make
+}
+
+check() {
+  cd "varnish-$pkgver"
+
+  make check
+}
```

Comment:
Is there a way to add a bit of `make` parallelism based on the number of packages? Serializing the build like this will take two needless eternities to complete.

Reply:
it's done autmatically, `make` is actually `make -j`nproc`

Reply:
well, technically, it depends on `MAKEFLAGS` in `/etc/makepkg.conf`, but the result is the same: it's not `PKGBUILD` responsibility to add `-j`